bundles/org.openhab.binding.squeezebox/README.md

```diff
@@ -25,7 +25,10 @@ Squeeze players may be official Logitech products or other players like [Squeeze
 ## Discovery
 
 A Squeeze Server is discovered through UPnP in the local network.
-Once it is added as a Thing the Squeeze Server bridge will discover Squeeze Players automatically.
+Once it is added as a Thing the Squeeze Server bridge will discover Squeeze Players automatically. If your Squeeze Server is not discovered automatically, you can add it manually by creating a .thing file containing something like this (more example [below](https://www.openhab.org/addons/bindings/squeezebox/#thing-configuration)):
```

Comment:
I find this change a little confusing.

- This section is talking about Discovery, so it seems out of place to be giving examples of how to create the thing using a `.things` file (not a `.thing` file). I'd suggest putting the `.things` file example further down in the document where the Items file example is given.

- Also, even if the case where server thing is not discovered automatically, it still can be created through the Admin UI (as all other things can be created). However, since this is not mentioned, it leaves the user with the idea that if it's not discovered automatically, then they need to create it using a `.things` file. It just feels like a discussion of how to create things manually using any method (UI or .things file) is out of place in the section on Discovery.

Reply:
@mhilbush I pretty much agree with your comments here - such a paragraph simply does not belong into the discovery section, especially as it is biased towards textual configuration.
I don't see this answered or addressed - what made you approve it now nonetheless?

Reply:
The example was removed, which is what I thought was most out of place.

Also, the whole doc is a bit dated, as it's been modified and extended quite a few times (mostly by me). As I'm considering a complete rewrite after the 3.0 release, I didn't find the change too objectionable.

Reply:
> This section is talking about Discovery, so it seems out of place to be giving examples of how to create the thing using a .things file 

From a user perspective, it makes sense that a section about discovery also mentions what to do when discovery doesn't work. To just say that the server will be discovered automatically is missleading for users where autodiscovery - for whatever reason - doesn't work and they will appreciate information about how to proceed. I agree that the details don't need to be in the discovery section. It would work just as well to move the example to a different place and then refer to it, e.g. by saying something like "If the Server is not discovered automatically, it can be added manually via the Admin UI or by creating a .things file as described below."

An alternative would be to rename the section from "Discovery" to something like "Adding the server". Autodiscovery, UI, and things file would then three ways of adding the server.

>if the case where server thing is not discovered automatically, it still can be created through the Admin UI (as all other things can be created). However, since this is not mentioned, it leaves the user with the idea that if it's not discovered automatically, then they need to create it using a .things file.

Fully agree. The UI option should also be mentioned. I didn't do so because the entire page seemed to be focussed on textual configuration and because I am not sure what UI changes are forthcoming in OH3.

Reply:
> From a user perspective, it makes sense that a section about discovery also mentions what to do when discovery doesn't work. 

But it does not make much sense to have such an explanation 200 times in 200 different add-ons. The suggestion here is simply "if discovery does not work, create the thing manually". Yeah, that applies in any case and is not specific to this binding. Therefore it should be part of the general tutorial and not part of each and every binding documentation.

Reply:
I see your point. Your aim is to keep the documentation as compact as possible. But can I ask you: what is the problem with giving users the same information on multiple pages (as long as it is relevant on that page, of course)? 

Nobody reads the documentation for each of the 200 add-ons and gets annoyed, like: "Why do they keep mentioning this?" I believe the typical way these pages are used is that you are unsure about how a particular addon works and are looking for information about how to proceed. That's why I thought it makes sense to provide that information right there where people are looking for it.

I realize that your point may also be that users should read the general tutorial before reading the documentation, and for those users who learn a new system in abstract terms before applying that knowledge in practice, that will work fine. But I think there are many users who find it easier to understand how a new system works by trying out how specific things work and building a more abstract picture from that. So to accommodate those users, I would think it makes sense for the documentation to also support that learning path. Perhaps their learning could be enhanced by adding yet another bit of information: "**As is the case with all add-ons**, if auto-discovery fails, the server can be added manually via the Admin UI or by creating a .things file as described below."